src/test/ui/matches_macro_not_in_the_prelude.stderr

```diff
@@ -0,0 +1,8 @@
+error: cannot find macro `matches` in this scope
```

Comment:
Please move all the related tests into their own directory.

Reply:
What directory? I looked for a `prelude` directory and didn’t find one. Or do you mean one just for this macro?

Reply:
Yeah just one for the macro to keep things organised.